README.md

```diff
@@ -8,46 +8,49 @@
+   embedded devices, desktops, mobile and web should be supported so that both the user and the developer feel comfortable on
+   their platform of choice.
 
-## Quick Start
+## Get Started
 
 Check the [online demos](#demos-running-in-webassembly-simulation) or
-[build on your own system](docs/building.md).
+[build on your own system](docs/building.md). Also check out the [documentation](#documentation).
 
 ## Current Status
 
-Slint is in active development. Its state of development depends on the kind of application.
+Slint is in active development. The state of the toolkit for each platform is as follows:
 
- - **Embedded**: *Ready!* Slint is already happily used by customers on embedded devices
+ - **Embedded**: *Ready!* Slint is already being used in production projects by customers 
    (running on an Arm processor with Linux).
- - **Microcontrollers**: *Ready*. You can make GUI working on MCU with less than 300K of RAM.
+ - **Microcontrollers** (MCU): *Ready*. Slint apps can run on MCUs with less than 300K of RAM.
    [Video of Slint on STM32](https://www.youtube.com/watch?v=NNNOJJsOAis) –
    [Video of Slint on RP2040](https://www.youtube.com/watch?v=dkBwNocItGs) – 
    [docs](https://slint-ui.com/docs/rust/slint/docs/mcu/index.html)
- - **Desktop**: *In Progress*. Slint is usable on Windows, Linux and Mac. We plan on building
-   up the desktop support in subsequent releases to reach the level of integration we're aiming for.
+ - **Desktop**: *In Progress*. While Slint is usable on Windows, Linux and Mac, we are working on
+    improving the platform support in subsequent releases to reach the level of integration that
+    have aimed for.
  - **Mobile** (Android/iOS): *Todo*. We haven't started supporting mobile platforms yet, but it
-   is our intention to do so in the future.
- - **Web**: Our focus is on Embedded and Desktop applications. We do not intend to compete in
-   the web space as a primary platform. That said, Slint can compile to WebAssembly and the
-   demos are running in a web browser. But the web support is currently limited to demo purposes.
- - **Accessibility**: Slint supports keyboard based navigation of many widgets, and user interfaces
-   are scalable. The basic infrastructure for assistive technology like screen readers is in place,
-   but requires the Qt backend at this time.
+   is our intention to do so in the near future.
+ - **Web**: *In Progress*. Slint apps can be compiled to WebAssembly and can run in a web browser.
+   As there are many other web frameworks, the web platform is not one of our primary target platforms.
+   The web support is currently limited to demo purposes.
 
-   We are aware that more work is needed to get best-of-class support for users with special needs.
+### Accessibility
+ Slint supports keyboard based navigation of many widgets, and user interfaces are scalable. 
```

Comment:
IIRC the actual markup standard requires a blank line after a headline (even though none of the renderers seems to care;-).

Reply:
Done